apps/common/models/i-casework.js

```diff
@@ -34,25 +33,27 @@ module.exports = class CaseworkModel extends Model {
     return params;
   }
 
-  sign() {
-    const date = (new Date()).toISOString().split('T')[0];
-    return crypto.createHash('md5').update(date + config.icasework.secret).digest('hex');
-  }
-
-  save() {
-    return Promise.resolve(this.prepare()).then(formData => {
-      const options = this.requestConfig({});
-      options.form = formData;
-      options.method = 'POST';
+  async save() {
+    return Promise.resolve(this.prepare()).then(async data => {
+      const params = {
+        url: this.url(),
+        data,
+        timeout: config.icasework.timeout,
+        method: 'POST'
+      };
```

Comment:
Again as above this could be wrapped in a try block with async await and provide error logging but leave that with me